asset/doc.css

```diff
@@ -5,7 +5,7 @@ div.odoc {
   position: relative;
 }
 
-div.odoc div.spec {
+div.odoc div.spec .spec.include {
```

Comment:
AFAICT, `div.spec` should work on `.spec.include`, unless the elements that are not `spec include` are not `div`? Why was this change necessary?

Reply:
It's mainly due to the fact that this issue inquired demarcating the include blocks, so I assumed this part doesn't work for the include blocks particularly, so it needed another CSS selector to detect. I'm sorry, I'll also redo this.

Reply:
That's a welcome change, I was just surprised we needed this, because the `.spec.include` should already be a subset of `div.spec`, but if that's not the case, let's keep this change.

Reply:
When I simulated it, it looked like it needed and I tried something like with `div.spec`, it didn't detect well.. Shall we also remove this and see how it goes?